 2019-05-07 11:17:38,059 [P11816/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 11:17:38,102 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 11:17:38,138 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 35ms)
 2019-05-07 11:17:38,365 [P11816/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 11:17:38,368 [P11816/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 11:17:38,371 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:17:38,842 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 471ms)
 2019-05-07 11:17:38,843 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 11:17:38,856 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 12ms)
 2019-05-07 11:17:38,880 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:17:38,881 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:17:38,881 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 11:17:38,890 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 8ms)
 2019-05-07 11:17:38,891 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:17:38,891 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:17:38,891 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 11:17:38,902 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 10ms)
 2019-05-07 11:17:38,920 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:17:38,921 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:17:38,921 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 11:17:38,928 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 7ms)
 2019-05-07 11:17:38,929 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:17:38,930 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:17:38,930 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 11:17:38,942 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 11ms)
 2019-05-07 11:17:41,259 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6730f0f6-f7c0-463a-8dc9-04d8e892db17
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:41,260 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:41,449 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 11:17:41,557 [P11816/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 107ms)
 2019-05-07 11:17:42,004 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ad5251d6-5c61-4b35-8ddf-5740f41ed5e9
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:42,004 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:43,290 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:2dbc0ee1-f183-4076-9d3c-b75b518c33a3
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:43,292 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:44,596 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:64f37adc-cd90-434b-bb0b-0da9e5f22ecb
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:44,597 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:45,925 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:c0ba89dc-2740-4950-8ead-e1ee88e4f1d1
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:45,926 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:47,199 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:e047ad5e-47fd-4d80-ad53-a605fdb5819f
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:47,200 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:48,477 [P11816/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: tcp:uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:9cd09313-c189-4572-9a85-eaef6a677a7d
Error Number:40615,State:1,Class:14
 2019-05-07 11:17:48,478 [P11816/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:17:48,484 [P11816/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   ved Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   ved Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   ved Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-05-07 11:17:58,543 [P11816/D2/T10] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 11:17:58,546 [P11816/D2/T10] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 11:17:58,971 [P11816/D2/T10] INFO  Umbraco.Web.ExamineStartup - Complete (took 424ms)
 2019-05-07 11:17:58,972 [P11816/D2/T10] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 11:18:28,990 [P11816/D2/T11] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: InitializationError

_shutDownMessage=Initialization Error
HostingEnvironment initiated shutdown
CONFIG change
CONFIG change
CONFIG change
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.AppDomainShutdownTimerCallback(Object state)
   ved System.Threading.TimerQueueTimer.CallCallbackInContext(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.TimerQueueTimer.CallCallback()
   ved System.Threading.TimerQueueTimer.Fire()
   ved System.Threading.TimerQueue.FireNextTimers()
   ved System.Threading.TimerQueue.AppDomainTimerCallback(Int32 id)
 2019-05-07 11:18:56,056 [P11816/D3/T9] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 11:18:56,082 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 11:18:56,100 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Hash determined (took 18ms)
 2019-05-07 11:18:56,288 [P11816/D3/T9] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 11:18:56,291 [P11816/D3/T9] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 11:18:56,295 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:18:56,809 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 513ms)
 2019-05-07 11:18:56,809 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 11:18:56,818 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 8ms)
 2019-05-07 11:18:56,842 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:18:56,843 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:18:56,843 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 11:18:56,850 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 6ms)
 2019-05-07 11:18:56,851 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:18:56,851 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:18:56,851 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 11:18:56,861 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 9ms)
 2019-05-07 11:18:56,880 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:18:56,880 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:18:56,880 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 11:18:56,887 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 6ms)
 2019-05-07 11:18:56,888 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:18:56,889 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:18:56,889 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 11:18:56,897 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 8ms)
 2019-05-07 11:18:58,870 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:371b4bd4-698b-42b3-8d0e-8428356e183c
Error Number:40615,State:1,Class:14
 2019-05-07 11:18:58,873 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:18:59,006 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 11:18:59,023 [P11816/D3/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 16ms)
 2019-05-07 11:18:59,459 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:f6083b2f-9864-448e-86d5-3c39ecfaa4fa
Error Number:40615,State:1,Class:14
 2019-05-07 11:18:59,460 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:00,794 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:38e84c07-588e-4088-bce4-e2f51ae5b3cb
Error Number:40615,State:1,Class:14
 2019-05-07 11:19:00,795 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:02,065 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:511ce986-18bc-4db5-9140-d42fade11f78
Error Number:40615,State:1,Class:14
 2019-05-07 11:19:02,066 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:03,372 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:4eaafc7c-d79f-4b30-b767-67e2cee0c624
Error Number:40615,State:1,Class:14
 2019-05-07 11:19:03,374 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:04,709 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:5e29c27e-f9c9-41c5-9b55-8e3a825479c5
Error Number:40615,State:1,Class:14
 2019-05-07 11:19:04,709 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:05,995 [P11816/D3/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net,1433, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:b9c827b4-e345-460d-8d8a-6a1076822368
Error Number:40615,State:1,Class:14
 2019-05-07 11:19:05,996 [P11816/D3/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:19:06,003 [P11816/D3/T9] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   ved Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   ved Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   ved Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-05-07 11:19:16,032 [P11816/D3/T11] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 11:19:16,033 [P11816/D3/T11] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 11:19:16,180 [P11816/D3/T11] INFO  Umbraco.Web.ExamineStartup - Complete (took 147ms)
 2019-05-07 11:19:16,182 [P11816/D3/T11] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 11:19:46,342 [P11816/D3/T12] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: InitializationError

_shutDownMessage=Initialization Error
HostingEnvironment initiated shutdown
CONFIG change
CONFIG change
CONFIG change
CONFIG change
CONFIG change
CONFIG change
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.AppDomainShutdownTimerCallback(Object state)
   ved System.Threading.TimerQueueTimer.CallCallbackInContext(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.TimerQueueTimer.CallCallback()
   ved System.Threading.TimerQueueTimer.Fire()
   ved System.Threading.TimerQueue.FireNextTimers()
   ved System.Threading.TimerQueue.AppDomainTimerCallback(Int32 id)
 2019-05-07 11:20:00,620 [P11816/D4/T9] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 11:20:00,658 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 11:20:00,680 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Hash determined (took 22ms)
 2019-05-07 11:20:00,802 [P11816/D4/T9] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 11:20:00,805 [P11816/D4/T9] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 11:20:00,816 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:20:01,608 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 791ms)
 2019-05-07 11:20:01,608 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 11:20:01,621 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 12ms)
 2019-05-07 11:20:01,653 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:20:01,655 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-05-07 11:20:01,655 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 11:20:01,667 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 11ms)
 2019-05-07 11:20:01,669 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:20:01,669 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:20:01,669 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 11:20:01,679 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 9ms)
 2019-05-07 11:20:01,702 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:20:01,702 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:20:01,702 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 11:20:01,715 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 12ms)
 2019-05-07 11:20:01,716 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:20:01,717 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:20:01,717 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 11:20:01,729 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 12ms)
 2019-05-07 11:20:04,256 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:ff2076c8-bafd-4236-89b4-2f43f34b2082
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:04,256 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:04,331 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 11:20:04,358 [P11816/D4/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 26ms)
 2019-05-07 11:20:04,720 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:53fe5fe4-5acd-4b21-9440-815b88875905
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:04,720 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:06,028 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:240c5634-4fa0-41e5-a1b2-ea682135bb1f
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:06,028 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:07,289 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:26e41349-1bb7-4c64-8386-7226acc1ba5c
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:07,291 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:08,555 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:1047a094-0d75-4ce4-9078-c186d7363a4a
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:08,556 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:09,942 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6022e86e-d2e9-4683-acd7-8737bfde0f8a
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:09,945 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:11,210 [P11816/D4/T9] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: uffechristiansen.database.windows.net, InitialCatalog: sdubackend. Exception: System.Data.SqlClient.SqlException (0x80131904): Cannot open server 'uffechristiansen' requested by the login. Client with IP address '185.19.132.85' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.
   ved System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   ved System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   ved System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   ved System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   ved System.Data.SqlClient.SqlConnection.Open()
   ved Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:b266792d-d432-40bc-bb3a-c469cd471413
Error Number:40615,State:1,Class:14
 2019-05-07 11:20:11,211 [P11816/D4/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-05-07 11:20:11,216 [P11816/D4/T9] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   ved Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   ved Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   ved Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   ved Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-05-07 11:20:21,367 [P11816/D4/T12] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 11:20:21,368 [P11816/D4/T12] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 11:20:21,370 [P11816/D4/T12] INFO  Umbraco.Web.ExamineStartup - Complete (took 2ms)
 2019-05-07 11:20:21,371 [P11816/D4/T12] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 11:20:51,510 [P11816/D4/T3] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: InitializationError

_shutDownMessage=Initialization Error
HostingEnvironment initiated shutdown
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.AppDomainShutdownTimerCallback(Object state)
   ved System.Threading.TimerQueueTimer.CallCallbackInContext(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.TimerQueueTimer.CallCallback()
   ved System.Threading.TimerQueueTimer.Fire()
   ved System.Threading.TimerQueue.FireNextTimers()
   ved System.Threading.TimerQueue.AppDomainTimerCallback(Int32 id)
 2019-05-07 11:21:00,497 [P11816/D5/T9] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 11:21:00,523 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 11:21:00,553 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Hash determined (took 30ms)
 2019-05-07 11:21:00,641 [P11816/D5/T9] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 11:21:00,643 [P11816/D5/T9] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 11:21:00,647 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:01,318 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 671ms)
 2019-05-07 11:21:01,318 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 11:21:01,319 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 1ms)
 2019-05-07 11:21:01,367 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:01,367 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:21:01,368 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 11:21:01,368 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2019-05-07 11:21:01,369 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:01,369 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:21:01,370 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 11:21:01,372 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2019-05-07 11:21:01,388 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:01,388 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:21:01,388 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 11:21:01,389 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2019-05-07 11:21:01,389 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:01,390 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:21:01,390 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 11:21:01,391 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 1ms)
 2019-05-07 11:21:04,983 [P11816/D5/T9] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-05-07 11:21:05,505 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 11:21:05,507 [P11816/D5/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 1ms)
 2019-05-07 11:21:05,623 [P11816/D5/T9] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-05-07 11:21:06,105 [P11816/D5/T9] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-05-07 11:21:06,273 [P11816/D5/T9] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-05-07 11:21:06,288 [P11816/D5/T9] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 5897ms)
 2019-05-07 11:21:06,295 [P11816/D5/T9] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-05-07 11:21:06,297 [P11816/D5/T9] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-05-07 11:21:06,680 [P11816/D5/T3] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57286/umbraco
 2019-05-07 11:21:06,681 [P11816/D5/T3] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57286/umbraco (UmbracoModule request)
 2019-05-07 11:21:07,501 [P11816/D5/T3] INFO  umbraco.content - Load Xml from file...
 2019-05-07 11:21:07,516 [P11816/D5/T3] INFO  umbraco.content - Loaded Xml from file.
 2019-05-07 11:21:09,381 [P11816/D5/T3] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:21:09,381 [P11816/D5/T3] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:21:09,381 [P11816/D5/T3] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-05-07 11:21:09,395 [P11816/D5/T3] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 13ms)
 2019-05-07 11:21:12,143 [P11816/D5/T3] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2019-05-07 11:22:07,608 [P11816/D5/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:22:07,609 [P11816/D5/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-05-07 11:22:07,609 [P11816/D5/T15] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-05-07 11:22:07,653 [P11816/D5/T15] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 43ms)
 2019-05-07 11:22:08,264 [P11816/D5/T16] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-05-06 11:22:07
 2019-05-07 11:26:00,255 [P11816/D5/T27] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:00,255 [P11816/D5/T27] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:00,256 [P11816/D5/T27] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-05-07 11:26:00,269 [P11816/D5/T27] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 13ms)
 2019-05-07 11:26:00,635 [P11816/D5/T26] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:00,636 [P11816/D5/T26] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-05-07 11:26:00,636 [P11816/D5/T26] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-05-07 11:26:00,650 [P11816/D5/T26] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 14ms)
 2019-05-07 11:26:09,579 [P11816/D5/T24] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username uffechristiansen@live.com from IP address ::1
 2019-05-07 11:26:09,587 [P11816/D5/T24] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: uffechristiansen@live.com logged in from IP address ::1
 2019-05-07 11:26:11,713 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:11,714 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:11,714 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-05-07 11:26:11,726 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 11ms)
 2019-05-07 11:26:11,762 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:11,763 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:11,763 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-05-07 11:26:11,763 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-05-07 11:26:11,784 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:11,785 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:11,785 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-05-07 11:26:11,796 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 10ms)
 2019-05-07 11:26:11,830 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:11,832 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-05-07 11:26:11,832 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-05-07 11:26:11,847 [P11816/D5/T24] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 14ms)
 2019-05-07 11:26:21,708 [P11816/D5/T33] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:21,709 [P11816/D5/T33] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:21,709 [P11816/D5/T33] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-05-07 11:26:21,725 [P11816/D5/T33] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 16ms)
 2019-05-07 11:26:27,875 [P11816/D5/T32] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 11:26:27,876 [P11816/D5/T32] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 11:26:27,876 [P11816/D5/T32] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-05-07 11:26:27,886 [P11816/D5/T32] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 10ms)
 2019-05-07 11:45:42,837 [P11816/D5/T63] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: uffechristiansen@live.com, IP Address: ::1
 2019-05-07 11:58:30,981 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-05-07 11:58:30,981 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-05-07 11:58:30,985 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-05-07 11:58:30,985 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-05-07 11:58:30,986 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-05-07 11:58:30,986 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-05-07 11:58:30,986 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-05-07 11:58:30,986 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-05-07 11:58:30,987 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-05-07 11:58:30,987 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-05-07 11:58:30,988 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-05-07 11:58:30,991 [P11816/D5/T133] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-05-07 11:58:30,993 [P11816/D5/T129] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-05-07 11:58:30,993 [P11816/D5/T130] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-05-07 11:58:30,993 [P11816/D5/T134] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-05-07 11:58:30,996 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-05-07 11:58:30,997 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-05-07 11:58:30,997 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-05-07 11:58:30,997 [P11816/D5/T124] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-05-07 11:58:31,000 [P11816/D5/T138] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-05-07 11:58:31,001 [P11816/D5/T138] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-05-07 11:58:31,072 [P11816/D5/T124] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 11:58:31,074 [P11816/D5/T124] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 11:58:31,078 [P11816/D5/T124] INFO  Umbraco.Web.ExamineStartup - Complete (took 3ms)
 2019-05-07 11:58:31,080 [P11816/D5/T124] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 11:59:01,140 [P11816/D5/T124] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: BinDirChangeOrDirectoryRename

_shutDownMessage=Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
HostingEnvironment initiated shutdown
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.OnCriticalDirectoryChange(Object sender, FileChangeEvent e)
   ved System.Web.FileChangesMonitor.OnCriticaldirChange(Object sender, FileChangeEvent e)
   ved System.Web.DirectoryMonitor.FireNotifications()
   ved System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)
   ved System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)
   ved System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   ved System.Threading.ThreadPoolWorkQueue.Dispatch()
   ved System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
 2019-05-07 12:01:01,260 [P11816/D6/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 12:01:01,279 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 12:01:01,298 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 18ms)
 2019-05-07 12:01:01,446 [P11816/D6/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 12:01:01,449 [P11816/D6/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 12:01:01,454 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:01:01,933 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 479ms)
 2019-05-07 12:01:01,933 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 12:01:01,943 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 9ms)
 2019-05-07 12:01:01,966 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:01:01,967 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:01:01,967 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 12:01:01,975 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 8ms)
 2019-05-07 12:01:01,977 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:01:01,977 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:01:01,977 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 12:01:01,991 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 13ms)
 2019-05-07 12:01:02,015 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:01:02,015 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:01:02,015 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 12:01:02,027 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 11ms)
 2019-05-07 12:01:02,028 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:01:02,029 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:01:02,029 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 12:01:02,041 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 11ms)
 2019-05-07 12:01:03,829 [P11816/D6/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-05-07 12:01:04,339 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 12:01:04,350 [P11816/D6/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 11ms)
 2019-05-07 12:01:04,413 [P11816/D6/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-05-07 12:01:04,753 [P11816/D6/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-05-07 12:01:04,873 [P11816/D6/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-05-07 12:01:04,886 [P11816/D6/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 3692ms)
 2019-05-07 12:01:05,084 [P11816/D6/T146] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57286/umbraco
 2019-05-07 12:01:05,086 [P11816/D6/T146] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57286/umbraco (UmbracoModule request)
 2019-05-07 12:01:46,752 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-05-07 12:01:46,753 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-05-07 12:01:46,763 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-05-07 12:01:46,764 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-05-07 12:01:46,765 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-05-07 12:01:46,765 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-05-07 12:01:46,765 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-05-07 12:01:46,765 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-05-07 12:01:46,766 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-05-07 12:01:46,766 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-05-07 12:01:46,769 [P11816/D6/T142] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 12:01:46,770 [P11816/D6/T146] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-05-07 12:01:46,771 [P11816/D6/T146] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-05-07 12:01:46,771 [P11816/D6/T142] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 12:01:46,771 [P11816/D6/T146] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-05-07 12:01:46,772 [P11816/D6/T146] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-05-07 12:01:46,772 [P11816/D6/T146] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-05-07 12:01:46,778 [P11816/D6/T142] INFO  Umbraco.Web.ExamineStartup - Complete (took 7ms)
 2019-05-07 12:01:46,779 [P11816/D6/T142] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 12:01:46,779 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-05-07 12:01:46,779 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-05-07 12:01:46,780 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-05-07 12:01:46,780 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-05-07 12:01:46,780 [P11816/D6/T142] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-05-07 12:01:46,793 [P11816/D6/T150] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-05-07 12:02:07,995 [P11816/D7/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 12:02:08,024 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 12:02:08,061 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 36ms)
 2019-05-07 12:02:08,224 [P11816/D7/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 12:02:08,227 [P11816/D7/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 12:02:08,231 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:08,734 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 503ms)
 2019-05-07 12:02:08,735 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 12:02:08,745 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 10ms)
 2019-05-07 12:02:08,767 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:08,768 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-05-07 12:02:08,768 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 12:02:08,776 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 7ms)
 2019-05-07 12:02:08,777 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:08,777 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:08,777 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 12:02:08,786 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 8ms)
 2019-05-07 12:02:08,808 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:08,808 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:08,809 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 12:02:08,816 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 7ms)
 2019-05-07 12:02:08,818 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:08,818 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:08,818 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 12:02:08,826 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 7ms)
 2019-05-07 12:02:10,741 [P11816/D7/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-05-07 12:02:11,138 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 12:02:11,150 [P11816/D7/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 12ms)
 2019-05-07 12:02:11,218 [P11816/D7/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-05-07 12:02:11,583 [P11816/D7/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-05-07 12:02:11,753 [P11816/D7/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-05-07 12:02:11,764 [P11816/D7/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 3840ms)
 2019-05-07 12:02:11,988 [P11816/D7/T150] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57286/umbraco
 2019-05-07 12:02:11,992 [P11816/D7/T150] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57286/umbraco (UmbracoModule request)
 2019-05-07 12:02:16,840 [P11816/D6/T12] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: BinDirChangeOrDirectoryRename

_shutDownMessage=Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
HostingEnvironment initiated shutdown
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Change in C:\Users\uffec\AppData\Local\Temp\Temporary ASP.NET Files\vs\ed616d9c\b28f830\hash\hash.web
Change in C:\Users\uffec\AppData\Local\Temp\Temporary ASP.NET Files\vs\ed616d9c\b28f830\hash\hash.web
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.OnCriticalDirectoryChange(Object sender, FileChangeEvent e)
   ved System.Web.FileChangesMonitor.OnCriticaldirChange(Object sender, FileChangeEvent e)
   ved System.Web.DirectoryMonitor.FireNotifications()
   ved System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)
   ved System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)
   ved System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   ved System.Threading.ThreadPoolWorkQueue.Dispatch()
   ved System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
 2019-05-07 12:02:19,453 [P11816/D7/T150] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username uffechristiansen@live.com from IP address ::1
 2019-05-07 12:02:19,461 [P11816/D7/T150] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: uffechristiansen@live.com logged in from IP address ::1
 2019-05-07 12:02:21,815 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:21,816 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:21,816 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-05-07 12:02:21,830 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 13ms)
 2019-05-07 12:02:21,941 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:21,942 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:21,942 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-05-07 12:02:21,953 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 10ms)
 2019-05-07 12:02:21,960 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:21,960 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:21,960 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-05-07 12:02:21,970 [P11816/D7/T130] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 10ms)
 2019-05-07 12:02:41,751 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:41,751 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:41,752 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-05-07 12:02:41,765 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 13ms)
 2019-05-07 12:02:42,151 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:02:42,151 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:02:42,152 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-05-07 12:02:42,170 [P11816/D7/T12] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 17ms)
 2019-05-07 12:03:12,270 [P11816/D7/T36] INFO  umbraco.content - Load Xml from file...
 2019-05-07 12:03:12,273 [P11816/D7/T36] INFO  umbraco.content - Loaded Xml from file.
 2019-05-07 12:03:12,458 [P11816/D7/T12] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-05-06 12:03:12
 2019-05-07 12:12:21,576 [P11816/D7/T110] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:12:21,578 [P11816/D7/T110] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-05-07 12:12:21,580 [P11816/D7/T110] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-05-07 12:12:21,594 [P11816/D7/T110] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 13ms)
 2019-05-07 12:12:23,787 [P11816/D7/T113] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:12:23,787 [P11816/D7/T113] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:12:23,787 [P11816/D7/T113] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-05-07 12:12:23,798 [P11816/D7/T113] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 10ms)
 2019-05-07 12:12:36,751 [P11816/D7/T107] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content '1' with Id '0' has been published.
 2019-05-07 12:12:38,154 [P11816/D7/T107] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:12:38,154 [P11816/D7/T107] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:12:38,155 [P11816/D7/T107] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-05-07 12:12:38,161 [P11816/D7/T107] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 5ms)
 2019-05-07 12:12:42,894 [P11816/D7/T111] INFO  umbraco.content - Save Xml to file...
 2019-05-07 12:12:42,920 [P11816/D7/T111] INFO  umbraco.content - Saved Xml to file.
 2019-05-07 12:12:52,564 [P11816/D7/T110] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content '2' with Id '0' has been published.
 2019-05-07 12:12:58,467 [P11816/D7/T117] INFO  umbraco.content - Save Xml to file...
 2019-05-07 12:12:58,484 [P11816/D7/T117] INFO  umbraco.content - Saved Xml to file.
 2019-05-07 12:13:51,658 [P11816/D7/T135] INFO  Umbraco.Core.MainDom - Stopping...
 2019-05-07 12:13:51,669 [P11816/D7/T135] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-05-07 12:13:51,723 [P11816/D7/T135] INFO  Umbraco.Web.ExamineStartup - Complete (took 54ms)
 2019-05-07 12:13:51,724 [P11816/D7/T135] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-05-07 12:13:51,726 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-05-07 12:13:51,726 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-05-07 12:13:51,726 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-05-07 12:13:51,727 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-05-07 12:13:51,727 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-05-07 12:13:51,727 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-05-07 12:13:51,727 [P11816/D7/T105] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-05-07 12:13:51,728 [P11816/D7/T107] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-05-07 12:13:51,728 [P11816/D7/T113] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-05-07 12:13:51,732 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-05-07 12:13:51,733 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-05-07 12:13:51,733 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-05-07 12:13:51,733 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-05-07 12:13:51,734 [P11816/D7/T116] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-05-07 12:13:51,734 [P11816/D7/T116] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-05-07 12:13:51,734 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-05-07 12:13:51,735 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-05-07 12:13:51,735 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-05-07 12:13:51,735 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-05-07 12:13:51,735 [P11816/D7/T135] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-05-07 12:13:51,736 [P11816/D7/T132] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-05-07 12:14:05,438 [P11816/D8/T94] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-NRI72M1
 2019-05-07 12:14:05,457 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-07 12:14:05,474 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Hash determined (took 17ms)
 2019-05-07 12:14:05,557 [P11816/D8/T94] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-07 12:14:05,560 [P11816/D8/T94] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-07 12:14:05,562 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:06,070 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 507ms)
 2019-05-07 12:14:06,071 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-07 12:14:06,079 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 8ms)
 2019-05-07 12:14:06,103 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:06,104 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:06,104 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-07 12:14:06,112 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 7ms)
 2019-05-07 12:14:06,113 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:06,113 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:06,113 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-07 12:14:06,122 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 8ms)
 2019-05-07 12:14:06,139 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:06,140 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:06,140 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-07 12:14:06,148 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 7ms)
 2019-05-07 12:14:06,150 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:06,150 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:06,150 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-07 12:14:06,158 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 8ms)
 2019-05-07 12:14:07,932 [P11816/D8/T94] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-05-07 12:14:08,722 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-05-07 12:14:08,737 [P11816/D8/T94] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 14ms)
 2019-05-07 12:14:08,949 [P11816/D8/T94] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-05-07 12:14:09,289 [P11816/D8/T94] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-05-07 12:14:09,372 [P11816/D8/T94] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-05-07 12:14:09,382 [P11816/D8/T94] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 4008ms)
 2019-05-07 12:14:09,520 [P11816/D8/T132] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:57286/umbraco
 2019-05-07 12:14:09,522 [P11816/D8/T132] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:57286/umbraco (UmbracoModule request)
 2019-05-07 12:14:21,834 [P11816/D7/T109] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: BinDirChangeOrDirectoryRename

_shutDownMessage=Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
HostingEnvironment initiated shutdown
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Ændringsbeskeder for kritiske mapper.
bin dir change or directory rename
Change in C:\Users\uffec\AppData\Local\Temp\Temporary ASP.NET Files\vs\ed616d9c\b28f830\hash\hash.web
Change in C:\Users\uffec\AppData\Local\Temp\Temporary ASP.NET Files\vs\ed616d9c\b28f830\hash\hash.web
HostingEnvironment forårsagede nedlukning

_shutDownStack=   ved System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   ved System.Environment.get_StackTrace()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   ved System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   ved System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   ved System.Web.HttpRuntime.OnCriticalDirectoryChange(Object sender, FileChangeEvent e)
   ved System.Web.FileChangesMonitor.OnCriticaldirChange(Object sender, FileChangeEvent e)
   ved System.Web.DirectoryMonitor.FireNotifications()
   ved System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)
   ved System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)
   ved System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   ved System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   ved System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   ved System.Threading.ThreadPoolWorkQueue.Dispatch()
   ved System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
 2019-05-07 12:14:28,333 [P11816/D8/T106] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:28,333 [P11816/D8/T106] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:28,334 [P11816/D8/T106] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-05-07 12:14:28,352 [P11816/D8/T106] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 17ms)
 2019-05-07 12:14:28,648 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:28,648 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:28,649 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-05-07 12:14:28,661 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 12ms)
 2019-05-07 12:14:28,945 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:14:28,946 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:14:28,946 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-05-07 12:14:28,958 [P11816/D8/T104] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 11ms)
 2019-05-07 12:15:09,742 [P11816/D8/T115] INFO  umbraco.content - Load Xml from file...
 2019-05-07 12:15:09,744 [P11816/D8/T115] INFO  umbraco.content - Loaded Xml from file.
 2019-05-07 12:15:09,763 [P11816/D8/T115] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-07 12:15:09,764 [P11816/D8/T115] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-07 12:15:09,764 [P11816/D8/T115] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-05-07 12:15:09,774 [P11816/D8/T115] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 10ms)
 2019-05-07 12:15:09,940 [P11816/D8/T108] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-05-06 12:15:09
